<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>PGCoE Network - Data & Visualization Tools</title>

  <link rel="stylesheet" href="css/styles.css">
  <link href="https://fonts.googleapis.com/css2?family=Lato&family=Raleway:wght@400;700&display=swap" rel="stylesheet">
  
  <style>
    #content {
      margin: 0;
      padding: 0;
    }

    .activePage {
      text-decoration: underline;
      font-weight: bold;
      color: #033C3F; 
      }
  </style>
</head>
  
<body>
  <nav>
    <a href="./">Home</a>
    <a href="./protocols.html">Sequencing Protocols</a>
    <a href="./workflows.html">Bioinformatics Workflows</a>
    <a href="./response.html">Response and Readiness</a>
    <a href="./dataviz.html"><span class="activePage">Data and Visualization</span></a>
    <a href="./education.html">Education</a>
  </nav>

  <div class="top-container">
      <div class="content">
        <h2>Data and Visualization Resources</h2>
        <p>In outbreak situations, it is often necessary to move quickly, especially to answer important questions about where an pathogen outbreak may be coming from, how it's spreading, and if the pathogen itself is changing as it spreads. To facilitate rapid response, the The Pathogen Genomics Centers of Excellence (PGCoE) Network is engaged in efforts to develop and share standardized databases and code repositories focused on analysis and visualization of pathogen genomic data and associated metadata.<p>
      </div>
      <div>
        <img src="img/pgcoe-network-graphic-element.png" alt="PGCoE logo">
      </div>
    </div>
  
    <div class="bottom-container">
      <h2>Interactive Datasets and Visualizations</h2>
      <div class="outer">
        <h3>State-Focused Nextstrain Builds from Washington State</h3>
        <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> <a href="https://nextstrain.org/groups/wadoh">State-focused builds</a> refer to Nextstrain analyses where data inclusion is designed to provide high visibility into disease dynamics in a jurisdiction of interest, such as a state. This sampling frame stands in contrast to the broader, representatively sampled datasets typically created by the Nextstrain team. The <a href="https://nextstrain.org/groups/wadoh">linked</a> builds are specific to Washington state, but can be forked to provide a template to users wishing to make their own Nextstrain analyses focusing on their particular jurisdiction of interest.</p>       
	  </div>
      <h2>Sampling Tools and Guidance</h2>
      <div class="outer">
        <h3>Work Instruction for Viral Variant Surveillance Sample Sizing Using <em>phylosamp</em></h3>
        <p><a href=""><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>This <a href="files/right_sizing_work_instruction.pdf">work instruction</a> provides a step-by-step guide on using the <a href="https://github.com/HopkinsIDD/phylosamp"><em>phylosamp</em> R package</a> to conduct sample sizing for viral variant surveillance while accounting for variant-specific genomic changes that can impact factors such as clinical test sensitivity, probablity of generating high-quality sequences, and probability of asymptomatic infection. Along with the step-by-step guide, <a href="files/right_sizing_examples.pdf">example analysis outputs</a> are provided to give an introduction into how these power analyses can be used to inform surveillance goals. An <a href="files/right_sizing_code_correctedformat.pdf">R code example</a> is also provided for ease of entry into coding for those not as familiar with the R environment.</p>
      </div>
    </div>

  <footer>
    <p>&copy; 2025 Pathogen Genomics Centers of Excellence</p>
  </footer>
    
</body>
</html>
