<body>
    <div class="top-container">
      <div class="content">
        <h2>Pathogen Sequencing Protocols</h2>
        <p>The Pathogen Genomics Centers of Excellence (PGCoE) Network is currently conducting a viral respiratory benchmarking study, which aims to compare the results of pathogen whole genome sequencing protocols used across the United States. When complete, the protocols listed below will be annotated according to their applicability for different sequencing platforms and effectiveness for specific pathogens. Further information on this benchmarking study, including details about the tool we are using to evaluate sequencing results, are available <a href="https://github.com/pgcoe/benchmarking">here</a>.</p>
      </div>
      <div>
        <img src="img/pgcoe-network-graphic-element.png" alt="PGCoE logo">
      </div>
    </div>
  
    <div class="bottom-container">
        <h2>Sample Collection Protocols</h2>
          <div class="outer">
            <h3>Sample Collection for Genomic Endpoints</h3>
            <p>The PGCoE network is developing a document that contains guidance for sample collection and storage with genomic endpoints in mind. This sample collection protocol will be linked here when available.</p>
          </div>

        <h2>Sequencing Primers and Probes</h2>
          <div class="outer">
              <h3>Primal Scheme Amplicon Sequencing Primers</h3>
              <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a> Primer sets for amplicon-based sequencing of many different pathogens (including RSV, SARS-CoV-2, measles, mpox, and others) are available on the <a href="https://labs.primalscheme.com/">labs.primalscheme site</a>, to which NE PGCoE partners regularly contribute.</p>
          </div>

        <h2>Viral Sequencing Protocols</h2>
          <div class="outer">
              <h3>Illumina Viral Surveillance Panel</h3>
              <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> Washington State Public Health Laboratory uses this protocol to conduct sequencing on all notifiable viral conditions in Washington State.</p>

              <h3>Automated Viral Amplicon Sequencing on Liquid Handlers</h3>
              <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a> <a href="https://github.com/DAMPLAB/Viral-Amplicon-Pipeline">This repository</a> contains protocols for automated viral sequencing on Hamilton Microlab STAR liquid handlers. These protocols have been validated by the DAMP Lab at Boston University.</p>
          </div>

        <h2>Bacterial Sequencing Protocols</h2>
          <div class="outer">
              <h3>Illumina Bacterial Surveillance Panel</h3>
              <p><a href="www.nwpage.org"></a><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> Washington State Public Health Laboratory uses this protocol to conduct sequencing on all notifiable viral conditions in Washington State.</p>
        </div>
</body>
