<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Splash template</title>

  <link rel="stylesheet" href="css/styles.css">
  <link href="https://fonts.googleapis.com/css2?family=Lato&family=Raleway:wght@400;700&display=swap" rel="stylesheet">
  
  <style>
    #content {
      margin: 0;
      padding: 0;
    }
  </style>
</head>
  
<body>
  <nav>
    <a href="./">Home</a>
    <a href="./protocols.html">Sequencing Protocols</a>
    <a href="./workflows.html">Bioinformatics Workflows</a>
    <a href="./response.html"><b><u>Response and Readiness</u></b></a>
  </nav>

  <div class="top-container">
      <div class="content">
        <h2>Responding to Emerging Pathogen Threats</h2>
        <p>The PGCoE Network also leads efforts to create the technical and social unification necessary to enable nimble, coordinated work between partners when generating and using sequence data as during an infectious disease event. This work ensures we have consistent ways of generating and capturing genomic and metadata, methods for sharing and/or collating across states, jurisdictions, and institutions, and protocols for how we communicate and work together during an infectious disease response.</p>
      </div>
      <div>
        <img src="img/pgcoe-network-graphic-element.png" alt="PGCoE logo">
      </div>
    </div>
  
    <div class="bottom-container">
        <h2>Sample Collection Protocols</h2>
          <div class="outer">
            <h3>Sample Collection for Genomic Endpoints</h3>
            <p>The PGCoE network is developing a document that contains guidance for sample collection and storage with genomic endpoints in mind. This sample collection protocol will be linked here when available.</p>
          </div>

        <h2>Sequencing Primers and Probes</h2>
          <div class="outer">
              <h3>Primal Scheme Amplicon Sequencing Primers</h3>
              <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a> Primer sets for amplicon-based sequencing of many different pathogens (including RSV, SARS-CoV-2, measles, mpox, and others) are available on the <a href="https://labs.primalscheme.com/">labs.primalscheme site</a>, to which NE PGCoE partners regularly contribute.</p>
          </div>

        <h2>Viral Sequencing Protocols</h2>
          <div class="outer">
              <h3>H5N1-specific Molecular Inversion Probes</h3>
              <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> Protocol for sequencing H5N1 viral genomes from animal specimens at Washington Animal Disease Diagnostic Laboratory using hybrid capture with Molecular Inversion Probes. This protocol will be linked here when available.</p>

              <h3>Automated Viral Amplicon Sequencing on Liquid Handlers</h3>
              <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a> <a href="https://github.com/DAMPLAB/Viral-Amplicon-Pipeline">This repository</a> contains protocols for automated viral sequencing on Hamilton Microlab STAR liquid handlers. These protocols have been validated by the DAMP Lab at Boston University.</p>
          </div>

        </div>

  <footer>
    <p>&copy; 2025 Pathogen Genomics Centers of Excellence</p>
  </footer>
    
</body>
</html>
