<body>
    <div class="top-container">
      <div class="content">
        <h2>Genomic Assembly and Analysis Workflows</h2>
        <p>The Pathogen Genomics Centers of Excellence (PGCoE) Network is currently conducting a viral respiratory benchmarking study, which aims to compare the results of pathogen genome assembly workflows used across the United States. When complete, the bioinformatics workflows listed below will be annotated according to their applicability for different sequencing platforms and effectiveness for specific pathogens.<p>
        <p>Further information on this benchmarking study, including details about the tool we are using to evaluate sequencing results, are available in our <a href="https://github.com/pgcoe/benchmarking">benchmarking repository</a>.</p>
      </div>
      <div>
        <img src="img/pgcoe-network-graphic-element.png" alt="PGCoE logo">
      </div>
    </div>
  
    <div class="bottom-container">
        <h2>Pathogen Genome Assembly Workflows</h2>
        <div class="outer">
            <h3>VAPER</h3>
            <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> <a href="https://nwpage.org/node/17">Pipeline</a> for assembling viral whole genome sequences from probe enrichment (a.k.a hybrid capture/enrichment), shotgun metagenomic, and tiled-amplicon sequence data.</p>
        </div>
        <div class="outer">
            <h3>viral-ngs</h3>
            <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a> A set of <a href="https://github.com/broadinstitute/viral-pipelines">scripts and tools</a> for the assembly and analysis of viral next generation sequencing data. All workflows are also available on the <a href="https://app.terra.bio/#workspaces/pathogen-genomic-surveillance/COVID-19">Terra platform.</a> </p>
        </div>

        <h2>Genomic Analysis Workflows</h2>
        <div class="outer">
            <h3>BigBacter</h3>
            <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> <a href="https://nwpage.org/tool-workflow/bigbacter">BigBacter</a> is a bacterial genomic surveillance pipeline to build phylogenetic trees, SNP matrices, and summary statistics, with output generated as MicroReact files. This pipeline also enables users to keep a running archive of genomic clusters that new sequences are added to if they're genomically similar, enabling longitudinal cluster monitoring.</p>
        </div>
      </div>
</body>
