<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>PGCoE Network - Bioinformatic Workflows</title>

  <link rel="stylesheet" href="css/styles.css">
  <link href="https://fonts.googleapis.com/css2?family=Lato&family=Raleway:wght@400;700&display=swap" rel="stylesheet">
  
  <style>
    #content {
      margin: 0;
      padding: 0;
    }

    .activePage {
      text-decoration: underline;
      font-weight: bold;
      color: #033C3F; 
      }
  </style>
</head>
  
<body>
  <nav>
    <a href="./">Home</a>
    <a href="./protocols.html">Sequencing Protocols</a>
    <a href="./workflows.html"><span class="activePage">Bioinformatics Workflows</span></a>
    <a href="./response.html">Response and Readiness</a>
    <a href="./dataviz.html">Data and Visualization</a>
    <a href="./education.html">Education</a>
  </nav>

  <div class="top-container">
      <div class="content">
        <h2>Genomic Assembly and Analysis Workflows</h2>
        <p>The Pathogen Genomics Centers of Excellence (PGCoE) Network is currently conducting a viral respiratory benchmarking study, which aims to compare the results of pathogen genome assembly workflows used across the United States. When complete, the bioinformatics workflows listed below will be annotated according to their applicability for different sequencing platforms and effectiveness for specific pathogens.<p>
        <p>Further information on this benchmarking study, including details about the tool we are using to evaluate sequencing results, are available in our <a href="https://github.com/pgcoe/benchmarking">benchmarking repository</a>.</p>
      </div>
      <div>
        <img src="img/pgcoe-network-graphic-element.png" alt="PGCoE logo">
      </div>
    </div>
  
    <div class="bottom-container">
      <h2>Pathogen Genome Assembly Workflows</h2>
      <div class="outer">
        <h3>TheiaCoV Workflow Series</h3>	
	      <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A <a href="https://theiagen.github.io/public_health_bioinformatics/latest/workflows/genomic_characterization/theiacov/">workflow series</a> for genomic characterization of SARS-CoV-2. There are currently five TheiaCoV workflows designed to accommodate different kinds of input data, including sequencing data generated on the Illumina, ONT, and ClearLabs platforms.</a></p>
        
        <h3>TheiaMeta</h3>	
	      <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A <a href="https://theiagen.github.io/public_health_bioinformatics/latest/workflows/genomic_characterization/theiameta/">workflow series</a> for genomic analysis from metagenomic sequencing, using both reference-based and reference-free approaches.</a></p>
        
        <h3>TheiaProk Workflow Series</h3>	
	      <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A <a href="https://theiagen.github.io/public_health_bioinformatics/latest/workflows/genomic_characterization/theiaprok/">workflow series</a> for genomic characterization of bacterial genomes. There are currently four TheiaProk workflows designed to accommodate different kinds of input data, including sequencing data generated on the Illumina and ONT platforms.</a></p>
        
        <h3>VAPER</h3>
        <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> <a href="https://nwpage.org/node/17">Pipeline</a> for assembling viral whole genome sequences from probe enrichment (a.k.a hybrid capture/enrichment), shotgun metagenomic, and tiled-amplicon sequence data.</p>       
        
        <h3>viral-pipelines</h3>
        <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A set of <a href="https://github.com/broadinstitute/viral-pipelines">scripts and tools</a> for the assembly and analysis of viral next generation sequencing data. All workflows are also available on the <a href="https://app.terra.bio/#workspaces/pathogen-genomic-surveillance/COVID-19">Terra platform.</a></p>
      
        <h3>virus-shotgun</h3>	
	      <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A <a href="https://github.com/neidl-connor-lab/virus-shotgun">simple pipeline</a> for metagenomic classification (using Kraken2), assembly, and variant calling from viral sequencing data.</a></p>
      </div>

      <h2>Genomic Analysis Workflows</h2>
      <div class="outer">
        <h3>BigBacter</h3>
        <p><a href="https://nwpage.org/"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NW.png"></a> <a href="https://nwpage.org/tool-workflow/bigbacter">BigBacter</a> is a bacterial genomic surveillance pipeline to build phylogenetic trees, SNP matrices, and summary statistics, with output generated as MicroReact files. This pipeline also enables users to keep a running archive of genomic clusters that new sequences are added to if they're genomically similar, enabling longitudinal cluster monitoring.</p>

        <h3>camlhmp</h3>
        <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-GA.png"></a>An <a href="https://github.com/rpetit3/camlhmp">infrastructure</a> for defining typing schemas for organisms. Current tools developed using this infrastructure include <a href="https://github.com/rpetit3/pasty?tab=readme-ov-file">pasty</a> (<em>Pseudomonas aeruginosa</em> serotyper), <a href="https://github.com/rpetit3/pbptyper">pbptyper</a> (<em>Streptococcus pneumoniae</em> penicillin binding protein typing), <a href="https://github.com/rpetit3/sccmec">sccmec</a> (Staphylcoccal chromosome cassette mec typing), and <a href="https://github.com/erinyoung/tulatyper">tulatyper</a> (<em>Francisella tularensis</em> subtyping), all of which are integrated with Bactopia.</a></p>

        <h3>Delphy</h3>
        <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A <a href="https://delphy.fathom.info/">fast, scalable, accurate, and accessible tool</a> for Bayesian phylogenetics with a drag-and-drop interface that requires no local installation or coding.</a></p>
        
        <h3>JUNIPER</h3>
        <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>A <a href="https://github.com/broadinstitute/juniper-analyses">framework</a> for predicting both phylogenies and transmission networks for viral outbreaks, accounting for both within-host variability and incomplete data.</a></p>
        
        <h3>reconstructR</h3>
        <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-NE.png"></a>An <a href="https://github.com/broadinstitute/reconstructR">R package</a> that allows for prediction of transmission events in infectious disease outbreaks and can account for within-host variability.</a></p>

        <h3>Tip Trait Association Testing (TTAT)</h3>
        <p><a href="https://nepgcoe.org/resources.html"><img style="display:inline-block; height:1em; width:auto; transform:translate(0, 0.1em)" src="img/icon_square-GA.png"></a>A <a href="https://github.com/leke-lyu/TTAT"> statistical framework</a> for testing if more closely related taxa on a phylogenetic tree are more likely to share the same trait values than we'd expect just by chance, allowing one to test for association between tree tips and chosen characteristics. There are also <a href="https://github.com/leke-lyu/jointEstimation">features</a> to analyze subregion heterogeneity of viral transmission and track when, where, and how many case introductions occur.</a></p>
      </div>
    </div>

  <footer>
    <p>&copy; 2025 Pathogen Genomics Centers of Excellence</p>
  </footer>
    
</body>
</html>
